test-app/src/message-board.ts

```diff
@@ -565,7 +576,7 @@ post.addField({
 
 api2.addQuery(
   "getPost",
-  getPostResolver.getField(api2, post.attribute(), {
+  getPost.getField(api2, post.attribute(), {
```

Comment:
What does it mean to add the resolver twice here? Its already added to `Query getPost` on api above.

Reply:
Not sure, I don't remember putting this here. I assume it would error. Should this be on api2?